helm/cluster-api-provider-cloud-director-app/values.yaml

```
@@ -0,0 +1,36 @@
+image:
+  name: quay.io/giantswarm/cluster-api-provider-cloud-director-vcd 
+  tag: 003336f
+
```

we use this for CAPO-App and CAPI-App to get the branch and commit sha filled out by architect during build time to have this information as label applied to all corresponding objects

```suggestion
project:
  branch: "[[ .Branch ]]"
  commit: "[[ .SHA ]]"
  
```